controller/internal/supervisor/iptablesctrl/iptables.go

```diff
@@ -275,11 +277,12 @@ func (i *iptables) DeleteRules(version int, contextID string, tcpPorts, udpPorts
 		zap.L().Error("unable to create cleanup configuration", zap.Error(err))
 		return err
 	}
-
+	markIntVal, _ := strconv.Atoi(mark)
```

Comment:
this function _has_ an error signature ...

Reply:
fixed